src/Symfony/Component/JsonPath/README.md

```diff
@@ -0,0 +1,13 @@
+JsonPath Component
+==================
+
+The JsonPath component eases JSON navigation using the JSONPath syntax as described in [RFC 9535](https://www.rfc-editor.org/rfc/rfc9535.html).
+
```

Comment:
could we also add a simple example of how to use the component?

Reply:
I see other components doing it in the README.md file directly

Reply:
Good idea, added a few lines and did the same as e.g. the RateLimiter README